images/ubuntu/templates/ubuntu-24.04.pkr.hcl

```diff
@@ -203,10 +208,10 @@ build {
     script          = "${path.root}/../scripts/build/configure-limits.sh"
   }
 
-  provisioner "file" {
-    destination = "${var.helper_script_folder}"
-    source      = "${path.root}/../scripts/helpers"
-  }
+  // provisioner "file" {
+  //   destination = "${var.helper_script_folder}"
+  //   source      = "${path.root}/../scripts/helpers"
+  // }
```

Comment:
Comments in the code are only needed to explain your actions or for the history. If you want to exclude the code from execution just remove it.

Reply:
Sure @Alexey-Ayupov , Thanks for your inputs.